Source/SaveExtension/Private/Serialization/SlotDataTask_Loader.cpp

```diff
@@ -368,8 +369,12 @@ void USlotDataTask_Loader::RespawnActors(const TArray<FActorRecord>& Records, co
 	// Respawn all procedural actors
 	for (const auto& Record : Records)
 	{
-		SpawnInfo.Name = Record.Name;
-		World->SpawnActor(Record.Class, &Record.Transform, SpawnInfo);
+		auto pExistingActor = StaticFindObjectFast(nullptr, SpawnInfo.OverrideLevel, Record.Name, true);
+		if (nullptr == pExistingActor)
```

Comment:
Pointers in UE4´s code convention dont use "p" prefix.

Also, this if would be simpler with `if (!pExistingActor) {}`

Reply:
Done, move exclude ActorToSpawn logic to PrepareLevel.